src/main/kotlin/dev/znci/rocket/commands/RocketCommand.kt

```diff
@@ -32,7 +33,8 @@ class RocketCommand(private val plugin: JavaPlugin) : TabExecutor {
         }
 
         val action = args[0].lowercase()
-        val scriptName = if (!args[1].endsWith(".lua")) "${args[1]}.lua" else args[1]
+        val rawScriptName = args[1]
```

Comment:
I think it would make more sense if `rawScriptName` and `scriptName` were flipped. `rawScriptName` should contain .lua, `scriptName` should not.